examples/simple/tests/ExampleTest.php

```diff
@@ -16,4 +15,37 @@ public function testItFails()
     {
         $this->assertTrue(false);
     }
+
+    /**
+     * @dataProvider exampleDataProvider
+     */
+    public function testWithDataProvider($actual, $expected)
+    {
+        $this->assertSame($actual, $expected);
+    }
+
+    protected function exampleDataProvider()
+    {
+        return [
+            [true, true],
+            [false, false],
+            [1, 1],
+            ['example', 'example'],
+        ];
+    }
+
+    /**
+     * @dataProvider exampleSecondDataProvider
+     */
+    public function testWithAnotherDataProvider($actual, $expected)
+    {
+        $this->assertSame($actual, $expected);
+    }
+
+    protected function exampleSecondDataProvider()
```

Comment:
Dataproviders need to be public
```suggestion
    public function exampleSecondDataProvider()
```